seed/main/trw.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
+<ldml>
+
+	<identity>
+
+		<version number="$new$"/>
+
+		<language type="trw"/>
+
+		<script type="Arab"/>
```

Comment:
drop `<script>` - not needed here. The script gets set somewhere else. This file is just `trw.xml` unless you expect Torwali is written with multiple scripts?